var/spack/repos/builtin/packages/py-click/package.py

```diff
@@ -1,17 +1,26 @@
-# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
 class PyClick(PythonPackage):
-    """A simple wrapper around optparse for powerful command line utilities."""
+    """Python composable command line interface toolkit."""
 
-    homepage = "http://github.com/mitsuhiko/click"
-    url = "https://pypi.io/packages/source/c/click/click-7.1.2.tar.gz"
+    homepage = "https://click.palletsprojects.com"
+    pypi = "click/click-7.1.2.tar.gz"
```

Comment:
```suggestion
    pypi = "click/click-7.1.2.tar.gz"
    url = "https://pypi.io/packages/source/c/click/click-7.1.2.tar.gz"
```
Still need the URL to actually build… :-/

This is one of the instances where upstream is ahead and sort-of diverged.

Reply:
I see. Didn't catch this because locally `spack install py-click@8.0.1` worked. Maybe because the sdist was already cached. Is there a way to ignore the cached files without deleting them by hand? This didn't work:
```
$ spack install --no-cache py-click@8.0.1
==> python@3.8.3 : has external module in ['python/3.8.3']
[+] /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/externals/2021-01-06/linux-rhel7-x86_64/gcc-9.3.0/python-3.8.3-suxrst (external python-3.8.3-qxs7hezm5wsyesn4ztcryy7zdd3ladi3)
[+] /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/libraries/2021-01-06/linux-rhel7-x86_64/gcc-9.3.0/py-setuptools-50.3.2-afe2g4
==> Installing py-click-8.0.1-dzw35prmqnk3op4hvwcfckwxyxkrncnu
==> Using cached archive: /gpfs/bbp.cscs.ch/home/sschmidt/.local/spack-software/.cache/_source-cache/archive/8c/8c04c11192119b1ef78ea049e0a6f0463e4c48ef00a30160c704337586f3ad7a.tar.gz
==> py-click: Executing phase: 'build'
==> py-click: Executing phase: 'install'
[+] /gpfs/bbp.cscs.ch/home/sschmidt/.local/spack-software/linux-rhel7-x86_64/gcc-9.3.0/py-click-8.0.1-dzw35p
```
it's still using the cached archive